--- resources/lib/Navigation.py.orig	2018-02-15 08:18:30.000000000 +0100
+++ resources/lib/Navigation.py	2018-02-15 11:35:32.565466257 +0100
@@ -415,6 +415,9 @@
         user_data = self._check_response(self.call_netflix_service({
             'method': 'get_user_data'}))
         if user_data:
+            self.kodi_helper.log('show_video_list list id for type ' +str(type) +' before refresh is ' + str(video_list_id))
+            video_list_id = self.refresh_list_id_for_type(type)
+            self.kodi_helper.log('show_video_list list id for type ' +str(type) +' after refresh is ' + str(video_list_id))
             for i in range(0, 4):
                 items = self._check_response(self.call_netflix_service({
                     'method': 'fetch_video_list',
@@ -479,6 +482,18 @@
                     build_url=self.build_url)
                 return listing
         return False
+    def refresh_list_id_for_type(self,type):
+        """The list_ids are not static so may need refreshed for example when stored as a widget"""
+        user_data = self._check_response(self.call_netflix_service({
+            'method': 'get_user_data'}))
+        video_list_ids = self._check_response(self.call_netflix_service({
+            'method': 'fetch_video_list_ids',
+            'guid': user_data['guid'],
+            'cache': True}))
+        if video_list_ids:
+            for video_list_id in video_list_ids['user']:
+                if video_list_ids['user'][video_list_id]['name'] == type:
+                    return video_list_id
 
     @log
     def show_profiles(self):
